devel.config

```diff
@@ -1,16 +1,21 @@
-backend_port = 7001
-frontend_port = 7002
+command: "run"
```

Comment:
Would it be possible to always take the command from the command line? Taking it commands from a config file is a bit weird, I think.
If not, no big deal.